### ISSUE TITLE
[Bug]: HTTP post transformation

### ISSUE THREAD

**Issue by vitalijs1** [apache/hop#2917]
### Apache Hop version?

2.4.0

### Java version?

java.version=11.0.10

### Operating system

Windows

### What happened?

there is one HTTP post transform that works fine in hop 2.1.0
but in version 2.4.0 it throws an error 500 "Internal Server error"

checked that Request entity field for api URL is the same in both versions

test hop pipeline attached:
[getSessionId.zip](https://github.com/apache/hop/files/11453320/getSessionId.zip)

### Issue Priority

Priority: 1

### Issue Component

Component: Actions, Component: API, Component: Hop Run, Component: Transforms

**Comment by sramazzina** [comment]
Hi @vitalijs1 
not clear to me if your example is a fully working example. I noticed that in your SOAP request there are username and password fields. Remember that we need a working example that we can use to reproduce the problem and take a look at your issue

**Comment by KBoersch** [comment]
I can confirm the problem. Until 2.2 I can send a POST request with HTTP POST. After that, until now (2.5) I can't. If I use the Rest Client transform instead, I'm able to send my POST request again.

I'm sending the request to a report-server, that expects the following json body:

{"submit":"submit",
"reportname":"Reportname",
"gitlab_branch":"MASTER",
"jobname":"2023-07-12-10-25"}

content-type json, that's it.

The body and content-type remains the same in rest client transform.

**Comment by hansva** [comment]
.take-issue

**Comment by hansva** [comment]
This was probably caused by enabling multipart on the data transfers, I changed this into an option this should resemble the old way of posting the requests more. It would be great if you could test this and provide feedback

**Comment by KBoersch** [comment]
Yeah sure. I will give you feedback, as soon as the new snapshot is out :)
